src/i18n/ko/nav.ts

```diff
@@ -14,4 +14,5 @@ export default NavDictionary({
 	'core-concepts/astro-components': '컴포넌트',
 	'core-concepts/astro-pages': '페이지',
 	'core-concepts/layouts': '레이아웃',
+	'core-concepts/framework-components': 'UI 프레임워크',
```

Comment:
```suggestion
	'core-concepts/framework-components': '프레임워크 컴포넌트',
```

일관된 용어(`프레임워크 컴포넌트`)로 내비게이션에 반영하는게 좋을 것 같습니다